timereport/app.py

```diff
@@ -124,6 +124,27 @@ def index():
         return ''
 
     if action == "list":
+        app.log.debug(f"Running list action with: {params}")
+        if 'total' in params:
```

Comment:
Might be dangerous to look for this value in the whole string.
If we know what position this argument should be in it's better to do `if params[1] == 'total':`... in my opinion